solcore/optics/beer_lambert.py

```diff
@@ -1,117 +1,161 @@
+        # absorbing light.
         elif type(layer_object) is TunnelJunction:
             n_layers_junction.append(len(layer_object))
             for i, layer in enumerate(layer_object):
                 widths.append(layer.width)
-                alphas.append(layer.material.alpha(wl_m))
+                alphas.append(layer.material.alpha(wavelength))
 
-        # For each junction, and layer within the junction, we get the absorption coeficient and the layer width.
+        # For each junction, and layer within the junction, we get the absorption
+        # coeficient and the layer width.
         elif type(layer_object) is Junction:
             n_layers_junction.append(len(layer_object))
-            kind = solar_cell[j].kind if hasattr(solar_cell[j], 'kind') else None
+            kind = solar_cell[j].kind if hasattr(solar_cell[j], "kind") else None
 
-            if kind == '2D':
-                # If the junction has a Jsc or EQE already defined, we ignore that junction in the optical calculation
-                if hasattr(solar_cell[j], 'jsc') or hasattr(solar_cell[j], 'eqe'):
-                    print('Warning: A junction of kind "2D" found. Junction ignored in the optics calculation!')
+            if kind == "2D":
+                # If the junction has a Jsc or EQE already defined, we ignore that
+                # junction in the optical calculation
+                if hasattr(solar_cell[j], "jsc") or hasattr(solar_cell[j], "eqe"):
+                    print(
+                        "Warning: A junction of kind '2D' found. Junction ignored in "
+                        "the optics calculation!"
+                    )
 
                     w = layer_object.width
 
                     def alf(x):
                         return 0.0 * x
 
                     solar_cell[j].alpha = alf
-                    solar_cell[j].reflected = interp1d(wl_m, solar_cell.reflected, bounds_error=False,
-                                                       fill_value=(0, 0))
+                    solar_cell[j].reflected = interp1d(
+                        wavelength,
+                        solar_cell.reflected,
+                        bounds_error=False,
+                        fill_value=(0, 0),
+                    )
 
                     widths.append(w)
-                    alphas.append(alf(wl_m))
+                    alphas.append(alf(wavelength))
 
-                # Otherwise, we try to treat is as a DB junction from the optical point of view
+                # Otherwise, we try to treat is as a DB junction from the optical point
+                # of view
                 else:
                     ASC.absorptance_detailed_balance(solar_cell[j])
                     w = layer_object.width
 
                     def alf(x):
-                        return -1 / w * np.log(np.maximum(1 - layer_object.absorptance(x), 1e-3))
+                        return (
```

Comment:
This is very minor and I guess it was done by black, I am fine with it, but the return here ooks a bit weird/harder to read than before? Anyway, if black will just keep changing it back then it will be pointless to do anything about it!